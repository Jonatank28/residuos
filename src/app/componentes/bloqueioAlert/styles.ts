import styled from 'styled-components/native';
import FeatherIcon from 'react-native-vector-icons/Feather';

export const FundoModal = styled.View`
  flex: 1;
  align-items: center;
  justify-content: center;
  position: relative;
  background-color: rgba(0, 0, 0, 0.5);
`;

export const FeatherIcone = styled(FeatherIcon)``;

export const ScrollContainer = styled.ScrollView.attrs({
  vertical: true,
  showsVerticalScrollIndicator: true,
  contentContainerStyle: {
    padding: 5,
  },
})``;

export const Container = styled.View`
  width: 80%;
  height: 30%;
  justify-content: center;
  align-items: center;
  border-radius: 10px;
  background-color: ${(props) => props.theme.card.background};
`;

export const IconContainer = styled.View`
  padding-top: 20px;
  padding-bottom: 20px;
  align-items: center;
  justify-content: center;
`;

export const Modal = styled.Modal``;

export const TituloContainer = styled.View`
  flex: 0.5;
  padding-top: 5px;
  width: 100%;
  flex-direction: row;
  align-items: center;
`;

export const DescricaoContainer = styled.View`
  flex: 1;
  justify-content: center;
  width: 100%;
  padding-right: 20px;
  padding-left: 20px;
`;

export const Titulo = styled.Text`
  flex: 1;
  text-align: center;
  color: ${(props) => props.theme.text.headline.color};
  font-weight: ${(props) => props.theme.text.headline.fontWeight};
  font-family: ${(props) => props.theme.text.headline.fontFamily};
  font-size: ${(props) => props.theme.text.headline.fontSize};
`;

export const Description = styled.Text`
  text-align: center;
  color: ${(props) => props.theme.text.headline.color};
  font-weight: ${(props) => props.theme.text.headline.fontWeight};
  font-family: ${(props) => props.theme.text.headline.fontFamily};
  font-size: ${(props) => props.theme.text.headline.fontSize};
`;
