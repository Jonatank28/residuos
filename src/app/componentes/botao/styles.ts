import styled from 'styled-components/native';
import FeatherIcon from 'react-native-vector-icons/Feather';

interface IContainer {
  backgroundColor?: string;
  hasBorder?: boolean;
  disabled?: boolean;
}


interface IText {
  textColor?: string;
}

export const Container = styled.TouchableOpacity<IContainer>`
  flex: 1;
  min-height: 50px;
  border-radius: 5px;
  position: relative;
  align-items: center;
  justify-content: center;
  border-width: ${(props) => `${props.hasBorder ? 1 : 0}px`};
  border-color: ${(props) => props.theme.border.color};
  background-color: ${(props) => props.disabled ? props.theme.border.color : props.backgroundColor ?? props.theme.button.background};
`;

export const FeatherIcone = styled(FeatherIcon)`
  position: absolute;
  top: 10px;
  left: 20px;
`;

export const TextoBotao = styled.Text<IText>`
  color: ${(props) => props.textColor ?? props.theme.text.button.color};
  font-weight: ${(props) => props.theme.text.button.fontWeight};
  font-family: ${(props) => props.theme.text.button.fontFamily};
  font-size: ${(props) => props.theme.text.button.fontSize};
`;
