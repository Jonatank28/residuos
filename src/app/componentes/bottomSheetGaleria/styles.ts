import styled from 'styled-components/native';

export const TituloSheetContainer = styled.TouchableOpacity`
  padding: 5px 0px;
  margin-bottom: 15px;
  flex-direction: row;
  align-items: center;
  justify-content: space-between;
`;

export const BottomSheetItem = styled.TouchableOpacity`
  padding: 18px;
  border-radius: 5px;
  margin-bottom: 10px;
  border-width: 1px;
  border-color: ${props => props.theme.card.border};
  background-color: ${props => props.theme.card.background};
`;

export const Titulo = styled.Text`
  color: ${(props) => props.theme.text.headline.color};
  font-weight: ${(props) => props.theme.text.headline.fontWeight};
  font-family: ${(props) => props.theme.text.headline.fontFamily};
  font-size: ${(props) => props.theme.text.headline.fontSize};
`;

export const BottomSheetItemTexto = styled.Text`
  color: ${(props) => props.theme.text.body1.color};
  font-weight: ${(props) => props.theme.text.body1.fontWeight};
  font-family: ${(props) => props.theme.text.body1.fontFamily};
  font-size: ${(props) => props.theme.text.body1.fontSize};
`;