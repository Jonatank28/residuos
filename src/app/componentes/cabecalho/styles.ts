import styled from 'styled-components/native';
import FeatherIcon from 'react-native-vector-icons/Feather';
import AwesomeIcon from 'react-native-vector-icons/FontAwesome';

interface IContainer {
  hasBorderBottom?: boolean;
  background?: string;
}

export const Container = styled.View<IContainer>`
  flex-direction: row;
  align-items: center;
  width: 100%;
  border-bottom-color: ${(props) => props.theme.border.color};
  border-bottom-width: ${(props) => (props.hasBorderBottom ? '1px' : '0px')};
  background-color: ${(props) => props.background ?? props.theme.primary};
`;

export const OfflineContainer = styled.View`
  height: 4px;
  background-color: ${(props) => props.theme.colors.orange};
`;

export const FeatherIcone = styled(FeatherIcon)``;

export const AwesomeIcone = styled(AwesomeIcon)``;

export const BotaoEsquerdoContainer = styled.TouchableOpacity`
  flex: 0.2;
  padding: 15px 0px 15px 0px;
  align-items: center;
`;

export const TituloContainer = styled.View`
  flex: 1;
`;

export const Titulo = styled.Text`
  text-align: center;
  color: ${(props) => props.theme.secundary};
  font-weight: ${(props) => props.theme.text.headline.fontWeight};
  font-family: ${(props) => props.theme.text.headline.fontFamily};
  font-size: ${(props) => props.theme.text.headline.fontSize};
`;

export const BotaoDireitoContainer = styled.View`
  flex: 0.2;
  padding: 15px 0px 15px 0px;
  align-items: center;
`;
