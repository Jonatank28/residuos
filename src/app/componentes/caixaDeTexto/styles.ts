import styled from 'styled-components/native';
import FeatherIcon from 'react-native-vector-icons/Feather';

interface IContainer {
  temErro: boolean;
  editavel?: boolean;
  backgroundColor?: string;
  borderLeftUp: number;
  borderRightUp: number;
  borderLeftDown: number;
  borderRightDown: number;
  marginTop: number;
  marginBottom: number;
  marginRight: number;
  marginLeft: number;
}

export const Container = styled.View<IContainer>`
  flex: 1;
  flex-direction: row;
  justify-content: center;
  align-items: center;
  min-height: 50px;
  padding-left: 10px;
  border-top-left-radius: ${(props) => `${props.borderLeftUp}px`};
  border-top-right-radius: ${(props) => `${props.borderRightUp}px`};
  border-bottom-left-radius: ${(props) => `${props.borderLeftDown}px`};
  border-bottom-right-radius: ${(props) => `${props.borderRightDown}px`};
  border-color: ${(props) => props.theme.colors.accent};
  border-width: ${(props) => `${props.temErro ? 1 : 0}px`};
  margin-top: ${(props) => `${props.marginTop}px`};
  margin-bottom: ${(props) => `${props.marginBottom}px`};
  margin-right: ${(props) => `${props.marginRight}px`};
  margin-left: ${(props) => `${props.marginLeft}px`};
  border-width: 1px;
  border-color: ${(props) => props.theme.border.color};
  background-color: ${(props) => !props.editavel ? '#F1f1f1' : props.backgroundColor ?? props.theme.card.background};
`;

export const FeatherIcone = styled(FeatherIcon)``;

export const Input = styled.TextInput`
  flex: 1;
  height: 40px;
  padding-left: 10px;
  width: 100%;
  color: ${(props) => props.theme.text.input.color};
  /* background-color: ${(props) => !props.editable ? props.theme.card.border : props.theme.text.input.background}; */
`;

export const ErroFormContainer = styled.View`
  align-items: flex-end;
  padding-right: 8px;
  margin-top: 5px;
`;

export const ErroFormTexto = styled.Text`
  color: ${(props) => props.theme.colors.accent};
  font-weight: ${(props) => props.theme.text.caption.fontWeight};
  font-family: ${(props) => props.theme.text.caption.fontFamily};
  font-size: ${(props) => props.theme.text.caption.fontSize};
`;
