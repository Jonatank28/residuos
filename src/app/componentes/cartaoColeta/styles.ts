import styled from 'styled-components/native';
import FeatherIcon from 'react-native-vector-icons/Feather';

interface IContainer {
  isDetails?: boolean;
}

interface IImage {
  width?: number;
  height?: number;
}

interface IRowContainer {
  flex?: number;
  isBottom?: boolean;
  justifyContent?: string;
  marginTop?: number;
  marginBottom?: number;
  marginLeft?: number;
  marginRight?: number;
}

interface IOpcaoContainer {
  backgroundColor?: string;
  hasBorder?: boolean;
}

interface ITitulo {
  color?: string;
  fontSize?: number;
}

export const Container = styled.TouchableOpacity<IContainer>`
  flex-direction: row;
  margin-bottom: 10px;
  border-radius: ${props => `${props.isDetails ? 5 : 0}px`};
  background-color: ${props => props.theme.card.background};
`;

export const FeatherIcone = styled(FeatherIcon)`
  color: ${props => props.theme.secundary};
`;

export const StatusTextoContainer = styled.View`
  align-items: flex-end;
  padding-bottom: 10px;
  padding-right: 10px;
  display: flex;
  flex-direction: row;
  justify-content: space-between;
  padding-left: 10;
`;

export const OpcaoContainer = styled.TouchableOpacity<IOpcaoContainer>`
  padding: 5px;
  margin: 0px 2px;
  border-radius: 5px;
  max-height: 25px;
  align-items: center;
  justify-content: center;
  background-color: ${props => props.backgroundColor ?? props.theme.colors.orange};
`;

export const StatusContainer = styled.View<IOpcaoContainer>`
  width: 10px;
  border-top-left-radius: ${props => `${props.hasBorder ? 5 : 0}px`};
  border-bottom-left-radius: ${props => `${props.hasBorder ? 5 : 0}px`};
  background-color: ${props => props.backgroundColor ?? props.theme.colors.green};
`;

export const ConteudoContainer = styled.View`
  flex: 1;
`;

export const Image = styled.Image`
  width: 100%;
  height: 100%;
`;

export const ImageContainer = styled.View<IImage>`
  width: ${props => `${props.width ?? 50}px`};
  height: ${props => `${props.height ?? 50}px`};
  margin-right: 5px;
`;

export const TituloContainer = styled.View`
  flex: 1;
`;

export const DescricaoPendenteContainer = styled.View`
  justify-content: center;
`;

export const DataContainer = styled.View`
  flex: 1;
  align-items: flex-end;
`;

export const RowContainer = styled.View<IRowContainer>`
  flex: ${props => props.flex ?? 1};
  justify-content: ${props => props.justifyContent ?? 'flex-start'};
  padding-right: ${props => `${props.marginRight ?? 10}px`};
  padding-left: ${props => `${props.marginLeft ?? 10}px`};
  padding-top: ${props => `${props.marginTop ?? 10}px`};
  padding-bottom: ${props => `${props.marginBottom ?? 10}px`};
  flex-direction: row;
  align-items: ${props => (props.isBottom ? 'center' : 'flex-start')};
`;

export const DescricaoTextoContainer = styled.View`
  flex: 1;
  flex-direction: row;
`;

export const RoteirizacaoContainer = styled.View`
  flex: 1;
  padding-left: 10px;
  padding-bottom: 5px;
`;

export const ColumnContainer = styled.View`
  flex: 1;
  padding: 0px 10px;
`;

export const HoraTexto = styled.Text<ITitulo>`
  color: ${props => props.theme.text.subhead.color};
  font-weight: ${props => props.theme.text.subhead.fontWeight};
  font-family: ${props => props.theme.text.subhead.fontFamily};
  font-size: ${props => (props.fontSize ? `${props.fontSize}px` : props.theme.text.subhead.fontSize)};
`;

export const DescricaoPendente = styled.Text<ITitulo>`
  text-transform: uppercase;
  color: ${props => props.color ?? props.theme.colors.accent};
  font-weight: ${props => props.theme.text.caption.fontWeight};
  font-family: ${props => props.theme.text.caption.fontFamily};
  font-size: ${props => props.theme.text.caption.fontSize};
`;

export const RoteirizacaoTexto = styled.Text`
  color: ${props => props.theme.primary};
  font-weight: ${props => props.theme.text.body1.fontWeight};
  font-family: ${props => props.theme.text.body1.fontFamily};
  font-size: ${props => props.theme.text.body1.fontSize};
`;

export const DescricaoTexto = styled.Text`
  color: ${props => props.theme.text.body1.color};
  font-weight: ${props => props.theme.text.body1.fontWeight};
  font-family: ${props => props.theme.text.body1.fontFamily};
  font-size: ${props => props.theme.text.body1.fontSize};
`;

export const EnderecoContainer = styled.View`
  flex: 1;
`;

export const OpcoesContainer = styled.View`
  flex-direction: row;
  z-index: 999;
`;

export const DescricaoTextoTitulo = styled.Text`
  padding-right: 5px;
  color: ${props => props.theme.text.body1.color};
  font-weight: 700;
  font-family: ${props => props.theme.text.body1.fontFamily};
  font-size: ${props => props.theme.text.body1.fontSize};
`;

export const TextoRowContainer = styled.View`
  flex-direction: row;
`;

export const OSTexto = styled.Text<ITitulo>`
  color: ${props => props.theme.text.headline.color};
  font-weight: ${props => props.theme.text.headline.fontWeight};
  font-family: ${props => props.theme.text.headline.fontFamily};
  font-size: ${props => (props.fontSize ? `${props.fontSize}px` : props.theme.text.headline.fontSize)};
`;

export const Titulo = styled.Text<ITitulo>`
  color: ${props => props.color ?? props.theme.text.headline.color};
  font-weight: ${props => props.theme.text.headline.fontWeight};
  font-family: ${props => props.theme.text.headline.fontFamily};
  font-size: ${props => (props.fontSize ? `${props.fontSize}px` : props.theme.text.headline.fontSize)};
`;
