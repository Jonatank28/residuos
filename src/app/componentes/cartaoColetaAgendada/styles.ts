import styled from 'styled-components/native';
import FeatherIcon from 'react-native-vector-icons/Feather';

interface IRowContainer {
  flex?: number;
  isBottom?: boolean;
  justifyContent?: string;
  marginTop?: number;
  marginBottom?: number;
  marginLeft?: number;
  marginRight?: number;
}

interface IOpcaoContainer {
  backgroundColor?: string;
}

interface ITitulo {
  color?: string;
  fontSize?: number;
}

export const Container = styled.TouchableOpacity`
  flex-direction: row;
  background-color: ${(props) => props.theme.card.background};
`;

export const FeatherIcone = styled(FeatherIcon)`
  color: ${(props) => props.theme.secundary};
`;

export const OpcaoContainer = styled.TouchableOpacity<IOpcaoContainer>`
  padding: 5px;
  margin: 0px 2px;
  border-radius: 5px;
  max-height: 25px;
  align-items: center;
  justify-content: center;
  background-color: ${(props) => props.backgroundColor ?? props.theme.colors.orange};
`;

export const StatusContainer = styled.View`
  width: 10px;
  background-color: ${(props) => props.theme.primary};
`;

export const ConteudoContainer = styled.View`
  flex: 1;
`;

export const PeriodicidadeContainer = styled.View`
  position: absolute;
  top: -10px;
  right: 10px;
  padding: 2px 4px;
  border-radius: 25px;
  background-color: ${(props) => props.theme.colors.green};
`;

export const Image = styled.Image`
  width: 100%;
  height: 100%;
`;

export const ImageContainer = styled.View`
  width: 45px;
  height: 45px;
  margin-right: 5px;
`;

export const TituloContainer = styled.View`
  flex: 1;
`;

export const DataContainer = styled.View`
  flex: 1;
  align-items: flex-end;
`;

export const RowContainer = styled.View<IRowContainer>`
  flex: ${(props) => props.flex ?? 1};
  justify-content: ${(props) => props.justifyContent ?? 'flex-start'};
  padding-right: ${(props) => `${props.marginRight ?? 10}px`};
  padding-left: ${(props) => `${props.marginLeft ?? 10}px`};
  padding-top: ${(props) => `${props.marginTop ?? 10}px`};
  padding-bottom: ${(props) => `${props.marginBottom ?? 10}px`};
  flex-direction: row;
  align-items: ${(props) => (props.isBottom ? 'center' : 'flex-start')};
`;

export const DescricaoTextoContainer = styled.View`
  flex: 1;
  flex-direction: row;
`;

export const RoteirizacaoContainer = styled.View`
  flex: 1;
  padding-left: 10px;
  padding-bottom: 5px;
`;

export const ColumnContainer = styled.View`
  flex: 1;
  padding: 0px 10px;
`;

export const HoraTexto = styled.Text<ITitulo>`
  color: ${(props) => props.theme.text.subhead.color};
  font-weight: ${(props) => props.theme.text.subhead.fontWeight};
  font-family: ${(props) => props.theme.text.subhead.fontFamily};
  font-size: ${(props) => (props.fontSize ? `${props.fontSize}px` : props.theme.text.subhead.fontSize)};
`;

export const RoteirizacaoTexto = styled.Text`
  color: ${(props) => props.theme.primary};
  font-weight: ${(props) => props.theme.text.body1.fontWeight};
  font-family: ${(props) => props.theme.text.body1.fontFamily};
  font-size: ${(props) => props.theme.text.body1.fontSize};
`;

export const DescricaoTexto = styled.Text`
  color: ${(props) => props.theme.text.body1.color};
  font-weight: ${(props) => props.theme.text.body1.fontWeight};
  font-family: ${(props) => props.theme.text.body1.fontFamily};
  font-size: ${(props) => props.theme.text.body1.fontSize};
`;

export const EnderecoContainer = styled.View`
  flex: 1;
`;

export const OpcoesContainer = styled.View`
  flex-direction: row;
  z-index: 999;
`;

export const DescricaoTextoTitulo = styled.Text`
  padding-right: 5px;
  color: ${(props) => props.theme.text.body1.color};
  font-weight: 700;
  font-family: ${(props) => props.theme.text.body1.fontFamily};
  font-size: ${(props) => props.theme.text.body1.fontSize};
`;

export const TextoRowContainer = styled.View`
  flex-direction: row;
`;

export const OSTexto = styled.Text<ITitulo>`
  color: ${(props) => props.theme.text.headline.color};
  font-weight: ${(props) => props.theme.text.headline.fontWeight};
  font-family: ${(props) => props.theme.text.headline.fontFamily};
  font-size: ${(props) => (props.fontSize ? `${props.fontSize}px` : props.theme.text.headline.fontSize)};
`;

export const Titulo = styled.Text<ITitulo>`
  color: ${(props) => props.color ?? props.theme.text.headline.color};
  font-weight: ${(props) => props.theme.text.headline.fontWeight};
  font-family: ${(props) => props.theme.text.headline.fontFamily};
  font-size: ${(props) => (props.fontSize ? `${props.fontSize}px` : props.theme.text.headline.fontSize)};
`;

export const PeriodicidadeTexto = styled.Text`
  color: ${(props) => props.theme.colors.white};
  font-weight: ${(props) => props.theme.text.caption.fontWeight};
  font-family: ${(props) => props.theme.text.caption.fontFamily};
  font-size: 10px;
`;
