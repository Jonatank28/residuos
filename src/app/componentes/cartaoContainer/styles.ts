import styled from 'styled-components/native';

export const Container = styled.View`
  margin-bottom: 10px;
  padding: 10px;
  border-radius: 5px;
  background-color: ${(props) => props.theme.card.background};
`;

export const RowContainer = styled.View`
  flex: 1;
  flex-direction: row;
  justify-content: space-between;
  align-items: center;
`;

export const TituloContainer = styled.View`
  padding: 5px 10px;
`;

export const DetalhesContainer = styled.View`
  padding: 5px;
`;

export const Titulo = styled.Text`
  color: ${(props) => props.theme.text.headline.color};
  font-weight: ${(props) => props.theme.text.headline.fontWeight};
  font-family: ${(props) => props.theme.text.headline.fontFamily};
  font-size: ${(props) => props.theme.text.headline.fontSize};
`;

export const Descricao = styled.Text`
  color: ${(props) => props.theme.text.body2.color};
  font-weight: ${(props) => props.theme.text.body2.fontWeight};
  font-family: ${(props) => props.theme.text.body2.fontFamily};
  font-size: ${(props) => props.theme.text.body2.fontSize};
`;
