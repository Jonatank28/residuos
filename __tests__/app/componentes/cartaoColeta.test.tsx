import React from 'react';
import renderer from 'react-test-renderer';

import { ThemeProvider } from 'styled-components/native';
import * as Themes from '../../../src/app/styles/themes';
import CartaoColeta from '../../../src/app/componentes/cartaoColeta';

describe('<CartaoColeta />', () => {
  beforeEach(() => {
    jest.resetAllMocks();
    jest.useFakeTimers();
  });

  afterEach(() => {
    jest.useRealTimers();
  });

  it('matches last snapshot details', () => {
    const tree = renderer.create(
      <ThemeProvider theme={Themes.light}>
        <CartaoColeta isDetails />
      </ThemeProvider>,
    );

    expect(tree.toJSON()).toMatchSnapshot();
  });

  it('matches last snapshot not details', () => {
    const tree: any = renderer.create(
      <ThemeProvider theme={Themes.light}>
        <CartaoColeta isDetails={false} />
      </ThemeProvider>,
    ).toJSON();

    expect(tree).toMatchSnapshot();
    expect(tree.children.length).toBe(2);
  });
});
