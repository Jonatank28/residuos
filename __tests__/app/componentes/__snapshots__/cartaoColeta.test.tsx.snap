// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CartaoColeta /> matches last snapshot details 1`] = `
<View
  accessible={true}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "backgroundColor": "#FFFFFF",
      "borderRadius": 5,
      "flexDirection": "row",
      "marginBottom": 10,
      "opacity": 1,
    }
  }
>
  <View
    backgroundColor="#FFF"
    hasBorder={true}
    style={
      Array [
        Object {
          "backgroundColor": "#FFF",
          "borderBottomLeftRadius": 5,
          "borderTopLeftRadius": 5,
          "width": 10,
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "flexBasis": 0,
          "flexGrow": 1,
          "flexShrink": 1,
        },
      ]
    }
  >
    <View
      marginBottom={2}
      style={
        Array [
          Object {
            "alignItems": "flex-start",
            "flexBasis": 0,
            "flexDirection": "row",
            "flexGrow": 1,
            "flexShrink": 1,
            "justifyContent": "flex-start",
            "paddingBottom": 2,
            "paddingLeft": 10,
            "paddingRight": 10,
            "paddingTop": 10,
          },
        ]
      }
    >
      <View
        style={
          Array [
            Object {
              "height": 50,
              "marginRight": 5,
              "width": 50,
            },
          ]
        }
      >
        <Image
          source={1}
          style={
            Array [
              Object {
                "height": "100%",
                "width": "100%",
              },
            ]
          }
        />
      </View>
      <View
        style={
          Array [
            Object {
              "flexBasis": 0,
              "flexGrow": 1,
              "flexShrink": 1,
            },
          ]
        }
      >
        <Text
          color="#FFF"
          style={
            Array [
              Object {
                "color": "#FFF",
                "fontFamily": "Proxima-Bold",
                "fontSize": 16,
                "fontWeight": "500",
              },
            ]
          }
        >
          [MISSING "EN.COMPONENTS.COLLECTCARD.DELIVERY" TRANSLATION]
        </Text>
        <View
          style={
            Array [
              Object {
                "flexDirection": "row",
              },
            ]
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#3C3C3C",
                  "fontFamily": "Proxima-Bold",
                  "fontSize": 16,
                  "fontWeight": "500",
                },
              ]
            }
          >
            [missing "en.components.collectCard.OS" translation]
          </Text>
          <Text
            color="#FFF"
            style={
              Array [
                Object {
                  "color": "#FFF",
                  "fontFamily": "Proxima-Bold",
                  "fontSize": 16,
                  "fontWeight": "500",
                },
              ]
            }
          />
        </View>
      </View>
      <View
        style={
          Array [
            Object {
              "alignItems": "flex-end",
              "flexBasis": 0,
              "flexGrow": 1,
              "flexShrink": 1,
            },
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#3C3C3C",
                "fontFamily": "Proxima-Bold",
                "fontSize": 16,
                "fontWeight": "500",
              },
            ]
          }
        >
          -
        </Text>
        <Text
          style={
            Array [
              Object {
                "color": "#515151",
                "fontFamily": "Proxima-Semibold",
                "fontSize": 12,
                "fontWeight": "400",
              },
            ]
          }
        >
          
        </Text>
      </View>
    </View>
  </View>
</View>
`;

exports[`<CartaoColeta /> matches last snapshot not details 1`] = `
<View
  accessible={true}
  focusable={false}
  onClick={[Function]}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "backgroundColor": "#FFFFFF",
      "borderRadius": 0,
      "flexDirection": "row",
      "marginBottom": 10,
      "opacity": 1,
    }
  }
>
  <View
    backgroundColor="#FFF"
    hasBorder={true}
    style={
      Array [
        Object {
          "backgroundColor": "#FFF",
          "borderBottomLeftRadius": 5,
          "borderTopLeftRadius": 5,
          "width": 10,
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "flexBasis": 0,
          "flexGrow": 1,
          "flexShrink": 1,
        },
      ]
    }
  >
    <View
      marginBottom={2}
      style={
        Array [
          Object {
            "alignItems": "flex-start",
            "flexBasis": 0,
            "flexDirection": "row",
            "flexGrow": 1,
            "flexShrink": 1,
            "justifyContent": "flex-start",
            "paddingBottom": 2,
            "paddingLeft": 10,
            "paddingRight": 10,
            "paddingTop": 10,
          },
        ]
      }
    >
      <View
        style={
          Array [
            Object {
              "height": 50,
              "marginRight": 5,
              "width": 50,
            },
          ]
        }
      >
        <Image
          source={1}
          style={
            Array [
              Object {
                "height": "100%",
                "width": "100%",
              },
            ]
          }
        />
      </View>
      <View
        style={
          Array [
            Object {
              "flexBasis": 0,
              "flexGrow": 1,
              "flexShrink": 1,
            },
          ]
        }
      >
        <Text
          color="#FFF"
          style={
            Array [
              Object {
                "color": "#FFF",
                "fontFamily": "Proxima-Bold",
                "fontSize": 16,
                "fontWeight": "500",
              },
            ]
          }
        >
          [MISSING "EN.COMPONENTS.COLLECTCARD.DELIVERY" TRANSLATION]
        </Text>
        <View
          style={
            Array [
              Object {
                "flexDirection": "row",
              },
            ]
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#3C3C3C",
                  "fontFamily": "Proxima-Bold",
                  "fontSize": 16,
                  "fontWeight": "500",
                },
              ]
            }
          >
            [missing "en.components.collectCard.OS" translation]
          </Text>
          <Text
            color="#FFF"
            style={
              Array [
                Object {
                  "color": "#FFF",
                  "fontFamily": "Proxima-Bold",
                  "fontSize": 16,
                  "fontWeight": "500",
                },
              ]
            }
          />
        </View>
      </View>
      <View
        style={
          Array [
            Object {
              "alignItems": "flex-end",
              "flexBasis": 0,
              "flexGrow": 1,
              "flexShrink": 1,
            },
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#3C3C3C",
                "fontFamily": "Proxima-Bold",
                "fontSize": 16,
                "fontWeight": "500",
              },
            ]
          }
        >
          -
        </Text>
        <Text
          style={
            Array [
              Object {
                "color": "#515151",
                "fontFamily": "Proxima-Semibold",
                "fontSize": 12,
                "fontWeight": "400",
              },
            ]
          }
        >
          
        </Text>
      </View>
    </View>
    <View
      style={
        Array [
          Object {
            "flexBasis": 0,
            "flexGrow": 1,
            "flexShrink": 1,
            "paddingBottom": 0,
            "paddingLeft": 10,
            "paddingRight": 10,
            "paddingTop": 0,
          },
        ]
      }
    >
      <View
        style={
          Array [
            Object {
              "flexBasis": 0,
              "flexDirection": "row",
              "flexGrow": 1,
              "flexShrink": 1,
            },
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#515151",
                "fontFamily": "Proxima-Semibold",
                "fontSize": 14.5,
                "fontWeight": "400",
              },
            ]
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#515151",
                  "fontFamily": "Proxima-Semibold",
                  "fontSize": 14.5,
                  "fontWeight": "700",
                  "paddingRight": 5,
                },
              ]
            }
          >
            [missing "en.components.collectCard.reason" translation]
          </Text>
           
        </Text>
      </View>
      <View
        style={
          Array [
            Object {
              "flexBasis": 0,
              "flexDirection": "row",
              "flexGrow": 1,
              "flexShrink": 1,
            },
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#515151",
                "fontFamily": "Proxima-Semibold",
                "fontSize": 14.5,
                "fontWeight": "400",
              },
            ]
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#515151",
                  "fontFamily": "Proxima-Semibold",
                  "fontSize": 14.5,
                  "fontWeight": "700",
                  "paddingRight": 5,
                },
              ]
            }
          >
            [missing "en.components.collectCard.fantasia" translation]
          </Text>
           
        </Text>
      </View>
    </View>
    <View
      isBottom={true}
      justifyContent="flex-end"
      marginBottom={5}
      marginTop={0}
      style={
        Array [
          Object {
            "alignItems": "center",
            "flexBasis": 0,
            "flexDirection": "row",
            "flexGrow": 1,
            "flexShrink": 1,
            "justifyContent": "flex-end",
            "paddingBottom": 5,
            "paddingLeft": 10,
            "paddingRight": 10,
            "paddingTop": 0,
          },
        ]
      }
    >
      <View
        style={
          Array [
            Object {
              "flexBasis": 0,
              "flexGrow": 1,
              "flexShrink": 1,
            },
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "#515151",
                "fontFamily": "Proxima-Semibold",
                "fontSize": 14.5,
                "fontWeight": "400",
              },
            ]
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#515151",
                  "fontFamily": "Proxima-Semibold",
                  "fontSize": 14.5,
                  "fontWeight": "700",
                  "paddingRight": 5,
                },
              ]
            }
          >
            [missing "en.components.collectCard.address" translation]
          </Text>
           
        </Text>
      </View>
      <View
        style={
          Array [
            Object {
              "flexDirection": "row",
              "zIndex": 999,
            },
          ]
        }
      >
        <View
          accessible={true}
          focusable={false}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#F91E1E",
              "borderRadius": 5,
              "justifyContent": "center",
              "marginBottom": 0,
              "marginLeft": 2,
              "marginRight": 2,
              "marginTop": 0,
              "maxHeight": 25,
              "opacity": 1,
              "paddingBottom": 5,
              "paddingLeft": 5,
              "paddingRight": 5,
              "paddingTop": 5,
            }
          }
        >
          <Text
            allowFontScaling={false}
            style={
              Array [
                Object {
                  "color": undefined,
                  "fontSize": 15,
                },
                Array [
                  Object {
                    "color": "#FFFFFF",
                  },
                ],
                Object {
                  "fontFamily": "Feather",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
                Object {},
              ]
            }
          >
            
          </Text>
        </View>
        <View
          accessible={true}
          focusable={false}
          onClick={[Function]}
          onResponderGrant={[Function]}
          onResponderMove={[Function]}
          onResponderRelease={[Function]}
          onResponderTerminate={[Function]}
          onResponderTerminationRequest={[Function]}
          onStartShouldSetResponder={[Function]}
          style={
            Object {
              "alignItems": "center",
              "backgroundColor": "#29D94C",
              "borderRadius": 5,
              "justifyContent": "center",
              "marginBottom": 0,
              "marginLeft": 2,
              "marginRight": 2,
              "marginTop": 0,
              "maxHeight": 25,
              "opacity": 1,
              "paddingBottom": 5,
              "paddingLeft": 5,
              "paddingRight": 5,
              "paddingTop": 5,
            }
          }
        >
          <Text
            allowFontScaling={false}
            style={
              Array [
                Object {
                  "color": undefined,
                  "fontSize": 15,
                },
                Array [
                  Object {
                    "color": "#FFFFFF",
                  },
                ],
                Object {
                  "fontFamily": "Feather",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
                Object {},
              ]
            }
          >
            
          </Text>
        </View>
      </View>
    </View>
  </View>
</View>
`;
